pkgs/development/compilers/dotnet/build-dotnet.nix

```diff
@@ -121,6 +122,24 @@ stdenv.mkDerivation (finalAttrs: rec {
       "aarch64-darwin" = "osx-arm64";
     };
 
+    updateScript =
+      if type != "sdk" then
```

Comment:
Why not add the update script only for `type == "sdk"` then? This currently prints the warning whenever I try to update GNOME with

```
nix-shell maintainers/scripts/update.nix --arg predicate '(path: pkg: pkg.updateScript.name or null == "gnome-update-script")'
```